website/docs/docs/running-a-dbt-project/using-the-dbt-ide.md

```diff
@@ -16,23 +16,23 @@ New dbt Cloud accounts will automatically be created with a Development Environm
 
 To create a Development Environment, navigate to the Environments page for your Project and click the "New Environment" button.
```

Comment:
question: is there a link we can use to get to this Environments page? it would greatly improve these navigation steps. right now, i can only get to it from the IDE

Reply:
@nghi-ly I don't see a way to link there directly. How is this?
```suggestion
To create a development environment, click **Deploy** and select **Environments**. Click **Create Environment**.
```